spec/forms/check_in_form_spec.rb

```diff
@@ -29,6 +29,21 @@
     end
   end
 
+  describe 'time field' do
+    it 'is invalid when not between 7am and 5pm' do
+      form = build :check_in_form, time: '1:00 AM'
```

Comment:
I can prevent this test from passing by changing this to `time: '6:59 AM CST'`. Should we include time zones whenever we specify times this way?

Reply:
It should match what form will actually put into the request. I just verified that with the current configuration, the Lolliclock widget doesn't attach a time zone. Here is a summary of the POST request:
```
Started POST "/check_ins" for ::1 at 2017-09-06 10:47:53 -0400
Processing by CheckInsController#create as HTML
  Parameters: {
    ...
    "check_in_form"=>{
      ...
      "day"=>"06 September, 2017",
      "time"=>" 9:47 AM",
      ...
    },
    ...
  }
```

Reply:
Right, I forgot that `day` and `time` here are strings. If I'm understanding it correctly, `timeliness` parses the begin and end of the allowed range as well as the user input using its default time zone, so to add the zone to the allowed range we'd have to alter the user input as well, making what you've done here the most general solution.